js/mixins/responsiveManagerMixin.js

```diff
@@ -56,6 +61,7 @@ var ResponsiveManagerMixin = {
     //set responsive data to state
     this.setState({
       componentWidth: componentWidth,
+      componentHeight: componentHeight > 0  ? componentHeight : 1,
```

Comment:
Because in the debugger all of this is one step, can you move "componentHeight > 0  ? componentHeight : 1" to a local variable so we know what the value is. It's easier to debug.